Assets/RGLUnityPlugin/Scripts/RadarSensor.cs

```diff
@@ -85,7 +87,8 @@ public void Awake()
                 .AddNodePointsCompact(CompactNodeId)
                 .AddNodeGaussianNoiseDistance(NoiseDistanceNodeId, 0, 0, 0)
                 .AddNodePointsTransform(ToRadarFrameId, Matrix4x4.identity)
-                .AddNodePointsRemoveGround(RemoveGroundNodeId, GroundAngleThreshold * Mathf.Deg2Rad, GroundDistanceThreshold, GroundFilterDistance)
+                .AddNodePointsFilterGround(FilterGroundNodeId, GroundAngleThreshold * Mathf.Deg2Rad)
+                .AddNodePointsCompactByField(CompactByFieldNodeId, RGLField.IS_GROUND_I32)
                 .AddNodePointsRadarPostprocess(RadarPostprocessNodeId, 0.1f, 0.1f);
```

Comment:
I wonder if the new ground filtering requires a point cloud in the sensor frame. If not we can reduce the pipeline by one points transform node:
1. `ToRadarFrameId` to be moved to the new subgraph (`rglSubgraphToRadarFrame`)
2. `ToWorldFrameId` to be removed. The point cloud in the world frame can be retrieved from `rglGraphRadar`